### ISSUE TITLE
test all site links

### ISSUE THREAD

**Issue by brunosardinepi** [brunosardinepi/pagefund#287]

**Comment by dmortazavi** [comment]
"Server Error (500)" when donating with a card that has an incorrect cc number. i was not logged in 

Active campaigns stopped showing up on the UAT Test Page

Tested all other links that are completed and received no errors

**Comment by dmortazavi** [comment]
Update - active campaigns started showing up again. weird

**Comment by brunosardinepi** [comment]
"Server Error (500)" when donating with a card that has an incorrect cc number

- By design. Stripe error: "Your card was declined. Your request was in test mode, but used a non test card."

**Comment by brunosardinepi** [comment]
i might have missed the point here. we'll have a custom error handler for this, but the overall behavior is expected

**Comment by brunosardinepi** [comment]
view all donations and see all active campaigns are dead links

**Comment by brunosardinepi** [comment]
@dmortazavi offloading to you pls

**Comment by dmortazavi** [comment]
Tested all site links and everything looks good. I was not able to test anything that involved banking information as I believe I should be waiting on that. only 2 comments below

- When inviting users, there is a ? that just reads, "some helpful info for the users"
- when on a page or campaign, if you click see all donations or campaigns, it just takes you to the top of the page. not sure if this is designed to be such with no donations or campaigns 
- When creating a Page, address is marked as optional but is required before clicking next. we talked about moving this to the bank act section so I'm assuming that is still in the works
